### ISSUE TITLE
[.5][ahdigital] Deposit (beta) Dropdown menus

### ISSUE THREAD

**Issue by wmbutler** [bitshares/bitshares-ui#1134]
- [ ] The dropdown menus should activate when the user clicks on any part of the menu including the right carat.
- [ ] Cursor should be a hand for the dropdown selectors

![deposit](https://user-images.githubusercontent.com/1254810/35983862-aca68e2e-0cb8-11e8-8dd4-641cc08eea00.gif)

**Comment by ahdigital** [comment]
Add me.

**Comment by ahdigital** [comment]
@wmbutler I've added the cursor hover effect for the select menus on deposit modal. Also, I've made sure the down arrows have the same cursor effect and that when clicked they open the select menu.

You will notice that the cursor is not always a hand on the first box, but this is a reasonable expectation in my opinion given that this field is a type ahead.

![select-hover-test](https://user-images.githubusercontent.com/3088454/36159985-ffcdaf06-10d7-11e8-8594-6e32c2135412.gif)
